content/node/leanpocket.md

```diff
@@ -0,0 +1,94 @@
+---
+title: LeanPocket
+menuTitle: LeanPocket
+weight: 35
+description: Run multiple nodes on the same hardware without scaling your resource requirements.
+---
+
+LeanPocket (sometimes stylized as "LeanPOKT") is a configuration that allows multiple Pocket nodes running on a single server to consolidate their resources, resulting in better scalability and reduced hardware costs.
+
+LeanPocket is a feature available from version 0.9.2 and above.
+
+## Background
+
+Pocket Core is a single-threaded application. This means that if you wanted to run multiple Pocket nodes on the same server, each node, running its own isolated process, would each use their own resources: RAM, disk space, network throughput, etc. As many aspects of nodes use shared resources (copies of the blockchain, most notably) this was deemed very inefficient.
```

Comment:
Pocket core is actually a multi-threaded application. However, Pocket core only allowed for one "node" per process historically.

Reply:
That's an important distinction, I'll definitely adjust the wording here.